Polyhedron/demo/Polyhedron/MainWindow.h

```diff
@@ -214,13 +214,13 @@ public Q_SLOTS:
   void throw_exception();
 
   /*!
-   * \brief setPolyhedronMode sets the global state of the demo to Polyhedron mode or Surface_mesh mode.
-   * In the Polyhedron_demo, a plugin that outputs a Facegraph_item will match the input type for the output.
-   * But when the input is not a Facegraph_item, the plugin will use this state to know what type to use.
-   * If isPolyhedronMode is true, the default output type will be Polyhedron (obviously), and if not, it will
-   * be Surface_mesh.
+   * \brief set_face_graph_default_type sets the global state of the application to `Polyhedron mode` or `Surface_mesh mode`.
+   * In the application, a plugin that outputs a `Facegraph_item` will match the input type for the output.
+   * But when the input is not a `Facegraph_item`, the plugin will use this state to know what type to use.
+   * If `isPolyhedronMode()` is `true`, the default output type will be `Polyhedron` (obviously), and if not, it will
+   * be `Surface_mesh`.
    */
-  void setPolyhedronMode(bool b);
+  void set_face_graph_default_type(bool b);
```

Comment:
I did not say it because it was kind of obvious for me: one the name is changed to `set_face_graph_default_type`, then the argument type cannot not be a `bool`: it has to be an enum, with two values with names like `POLYHEDRON_3` and `SURFACE_MESH` (the names of the enumerators have to be uppercase, by CGAL coding conventions).